tests/src/test/java/org/prevayler/implementation/JournalSerializerTest.java

```diff
@@ -102,12 +102,12 @@ public void testCompressedAndEncryptedJournal() throws Exception {
 
     startAndCrash("journal", des);
 
-    assertEquals("88;withQuery=false;systemVersion=1;executionTime=1000002\r\n" +
-        "E+\315\256D\023p\241\271hC\n\025\313n\340%\306:\273T\022\365\002\312/\2529'\247\224\220\217\367\006\306\353\n\363\004o\b\337\032\320\207\255\265\007\032\213\177_Wg\360X\366\214\316\276\366 \341!~!\035Y*\3473OB\007\"hmDT_\347\337\354\337\360\211\226\004IV.N\205\204\246\001k8K\007\205W\236\312EI\336\360\037w\177\255\034\035me3\311\374\223\212\377\256\021\f\367\272\341C\302\367\001d\314\333\r\n" +
-        "88;withQuery=false;systemVersion=2;executionTime=1000004\r\n" +
-        "E+\315\256D\023p\241\271hC\n\025\313n\340%\306:\273T\022\365\002\312/\2529'\247\224\220\217\367\006\306\353\n\363\004o\b\337\032\320\207\255\265\007\032\213\177_Wg\360X\366\214\316\276\366 \341!~!\035Y*\3473OB\007\"hmDT_\347\337\354\337\360\211\226\004IV.N\205\204\246\001k8K\007\205W\236\346\374\017a\205C|\272\032\237\226\307\26662h2\262\277\232\022B\022\377\341C\302\367\001d\314\333\r\n" +
-        "88;withQuery=false;systemVersion=3;executionTime=1000006\r\n" +
-        "E+\315\256D\023p\241\271hC\n\025\313n\340%\306:\273T\022\365\002\312/\2529'\247\224\220\217\367\006\306\353\n\363\004o\b\337\032\320\207\255\265\007\032\213\177_Wg\360X\366\214\316\276\366 \341!~!\035Y*\3473OB\007\"hmDT_\347\337\354\337\360\211\226\004IV.N\205\204\246\001k8K\007\205W\236+\026\216\030\322O\271\3627\360\020{\\5\031\201\223\212\377\256\021\f\367\272\341C\302\367\001d\314\333\r\n",
+    assertEquals("80;withQuery=false;systemVersion=1;executionTime=1000002\r\n" +
```

Comment:
You say the defaults have changed for DES, causing this size to change. Is there a way to make the defaults explicit, so that it behaves the same for all JDK versions?

Reply:
I tried to adjust Cipher static factory, but I couldn't. Maybe my teory is wrong, can you test new version in your machine?

Reply:
Hi @jsampson, did you try in your machine? I think that it was a security fix on jdk. (removed old ciphers).

Reply:
Thanks for your patience, and thanks again for this contribution. I got swamped at the end of the year, but found some time to investigate this today. I'll go ahead and merge this pull request, and I'll add a comment below explaining what I learned.